lib/query/lib/prepareForDelivery.js

```diff
@@ -150,7 +150,7 @@ export function storeOneResults(node, sameLevelResults) {
         _.each(sameLevelResults, result => {
             // The reason we are doing this is that if the requested link does not exist
             // It will fail when we try to get undefined[something] below
-            if (result === undefined) {
+            if (!result) {
```

Comment:
Just a newbie question, wouldn't `(result === undefined || result === null)` be better here so that we don't include `false` results into this, especially if null is the problem? Or is a boolean result here not possible.

Reply:
I'm assuming it would error as well with false, if you look at the stack trace I linked in the original post.